ONSdigital/response-operations-ui
# Sign out of user account on permission change

# What and why?
There is a requirement for the user to be signed out when the users permission has been changed. This code change should accomplish this.

# How to test?
1. In a browser profile (Chrome) sign in with an account you want to change the permissions for (test user)
2. In one browser profile sign in with an admin user
3. Using the admin account, goto manage users and select the test user that was used to sign in
4. Change the permissions (either ticket a new permission or untick a permission)
5. Click save changed
6. Go back to the test user and either refresh the screen or click on another tab
7. The user should be signed out a redirected to the sign in page

# Trello
https://trello.com/c/frmUHnXZ/1798-spike-sign-out-of-user-account-on-permission-change-1-2-days
